### DISCUSSION TITLE
how to add server arguments on mac?

### DISCUSSION THREAD

**Discussion by eyeweaver** [vladmandic/sdnext#505] [Q&A] [answered]
Hello..
I need to add these command line arguments to the launcher on Mac Os:
--share --listen --upgrade --enable-insecure --autolaunch --enable-insecure

in Automatic 1111 I used to edit the launcher file and add the arguments to it.. it it would be updated without resetting the settings..
but it's different here as with each update the Launcher file is reset to it's default status.

can someone please help me?
Thank you all in advance :)

**Comment by vladmandic** [comment, marked as answer]
create file `webui-user.sh`

```sh
./webui.sh --share --listen --upgrade --enable-insecure --autolaunch --enable-insecure
```

**Reply by eyeweaver** [reply]
Thank you so much for your help and great support..

**Comment by jm18919** [comment]
hi，I try to add --opt-sub-quad-attention but it seems that there isn't this option. But searching this repo's issues I coud find someone else has this option. I am quite confused. How to add this option?

**Reply by vladmandic** [reply]
It's in UI settimgs.
